docker/neutron/neutron-mlnx-agent/Dockerfile.j2

```diff
@@ -29,7 +29,8 @@ LABEL maintainer="{{ maintainer }}" name="{{ image_name }}" build-date="{{ build
     ] %}
```

Comment:
Is it also required for CentOS 8?

Reply:
CentOS8 only uses python3 right?

Reply:
There is https://centos.pkgs.org/8/epel-x86_64/python3-defusedxml-0.6.0-1.el8.noarch.rpm.html, but I'm guessing as they are using this on tripleo it works without it? Also that needs epel.